107 cleanup items

neal-timpe opened the pull request:
This PR addresses several 4.3 only issues. It should only be cherrypicked to the enterprise-4.3. 

It addresses three issues:
1. Updates OSSM version variable to 107
2. Removes a topic that was written for a specific OCP 4.1 to 4.2 and earlier version OSSM update scenario that is no longer relevant.
3. Updates some UI text for OCP 4.3.

Made an issue to track this work.
https://issues.redhat.com/projects/OSSMDOC/issues/OSSMDOC-45

JStickler commented:
LGTM (Changes should get CP into the 4.4 branch too!)

vikram-redhat commented:
@neal-timpe for the removal of the update topic, is that no longer relevant in 4.3/4.4? Do we need to send out updates and get ack from support/PM/QE?

neal-timpe commented:
@vikram-redhat QE is already on this PR and approved. I've also reached out to engineering.

neal-timpe commented:
@tvieira @JStickler I made a change since you approved. Instead of removing the topic, I removed the information about that 4.1->4.2 update and added some more general information about updates with operators. Can you take another look?

JStickler commented on service_mesh/service_mesh_install/updating-ossm.adoc:
```diff
@@ -4,42 +4,9 @@ include::modules/ossm-document-attributes.adoc[]
 
-.Prerequisites
+If you choose to update manually, the Operator Lifecycle Manager (OLM) controls the installation, upgrade, and role-based access control (RBAC) of Operators in a cluster. It is deployed by default in {product-title}.
+OLM uses CatalogSources, which use the Operator Registry API, to query for available Operators as well as upgrades for installed Operators.
```
"It" is ambiguous here, I'd change it to "OLM runs by default in {product-title}.

neal-timpe replied:
@JStickler Good catch. Totally forgot about the title when I copied it from the old file. Made those changes. Should be ready.

JStickler commented:
/cherry-pick enterprise-4.3

JStickler commented:
/cherry-pick enterprise-4.4
